src/tests/ftest/container/multiple_delete.py

```diff
@@ -46,25 +45,28 @@ def test_multiple_container_delete(self):
                 self.server_group, self.pool, self.container.uuid)
             # If the transfer size is less than 4K, the objects are
             # inserted into SCM and anything greater goes to SSD
-            self.run_ior_with_pool()
+            self.run_ior_with_pool(create_cont=False)
             self.container.destroy()
             scm_fs, ssd_fs = self.get_pool_space()
-            out.append("iter = {}, scm = {}, ssd = {}".format(
-                i+1, scm_fs, ssd_fs))
+            out.append("iter = {}, scm = {}, ssd = {}".format(i+1, scm_fs, ssd_fs))
 
         self.log.info("Initial Free Space")
-        self.log.info("SCM = %d, SSD = %d", initial_scm_fs, initial_ssd_fs)
+        self.log.info("SCM = %d, NVMe = %d", initial_scm_fs, initial_ssd_fs)
         self.log.info("Free space after each cont create/del iteration")
         self.log.info("\n".join(out))
         final_scm_fs, final_ssd_fs = self.get_pool_space()
         self.log.info("Final free Space after all iters")
-        self.log.info("SCM = %d, SSD = %d", final_scm_fs, final_ssd_fs)
+        self.log.info("SCM = %d, NVMe = %d", final_scm_fs, final_ssd_fs)
 
-        self.log.info("Verifying SSD space is recovered")
-        self.check_pool_free_space(self.pool, expected_nvme=initial_ssd_fs)
+        self.log.info("Verifying NVMe space is recovered")
+        try:
+            self.pool.check_free_space(expected_nvme=initial_ssd_fs)
```

Comment:
(pylint-no-member) Instance of 'list' has no 'check_free_space' member
(pylint-no-member) Instance of 'TestPool' has no 'check_free_space' member